Makefile

```diff
@@ -77,9 +77,9 @@ testcases:
 		--validate_out="lang=go:./go" \
 		`find . -name "*.proto"`
 
-validate/validate.pb.go:
+test-validate:
```

Comment:
This is a dependency of other targets, so I don't think you can just rename.

Reply:
We don't need this target if we commit the file. I updated the recipes to compare bazel output against committed file since protoc has slightly different output than bazel.